www/_template/getting-started.md

```diff
@@ -3,43 +3,278 @@ layout: layouts/main.njk
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="description" content="Starter Snowpack App" />
+    <title>Starter Snowpack App</title>
+  </head>
+  <body>
+    <h1>Welcome to Snowpack!</h1>
+  </body>
+</html>
+```
+
+Add the Snowpack development server to `package.json` under as the `start` script:
+
+```diff
+  "scripts": {
++   "start": "snowpack dev",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+
+```
 
-Here's a short list of what you can do with Snowpack:
+Run the following on the command line to start the Snowpack development server
+
+```
+npm run start
+```
+
+Snowpack displays the local host address and automatically opens the page in your default browser.
+
+IMAGE: GIF. Side by side of the terminal showing the dev server output. The dev server output displays the localhost address the project is running on. In a browser window you can see the running project on localhost, which is "Welcome to Snowpack" on a white background. An edit is made to `index.html` adding an exclamation point, the browser window shows the updated text as soon as the file is saved.
+
+Try changing the index.html and saving while the server is running, the site should refresh and show changes automatically.
+
+## Using ESM (ES Modules)
+
+You're ready to add some ES modules (ESM). JavaScript's native ES Module (ESM) syntax is the magic behind Snowpack's unbundled development. ESM lets you define explicit imports & exports that browsers and build tools can better understand and optimize for. If you're familiar with the `import` and `export` keywords in JavaScript, then you already know ESM!
+
+All modern browsers support ESM, so Snowpack is able to ship this code directly to the browser during development. This is what makes Snowpack's **unbundled development** workflow possible.
+
+Create a JavaScript file called `hello-world.js` that exports a `helloWorld` ES module:
+
+```js
+export function helloWorld() {
+  console.log('Hello World!');
+}
+```
+
+Create an `index.js` that imports your new module using ESM syntax:
+
+```js
+import { helloWorld } from './hello-world.js';
+
+helloWorld();
+```
+
+Finally, add your `index.js` to `index.html` at the bottom of the `<body>` tag
+
+```diff
+  <body>
+    <h1>Welcome to Snowpack!</h1>
++   <script type="module" src="/index.js"></script>
+  </body>
+```
+
+> 💡 Tip: Snowpack detects the files in `index.html` for processing
+
+Check your console on your Snowpack site. You should see "Hello World!" Try making a change to the module. Snowpack will rebuild that module without rebuilding the rest of your code. Snowpack builds **every file individually and caches it indefinitely.** Your development environment will never build a file more than once and your browser will never download a file twice (until it changes). This is the real power of unbundled development.
```

Comment:
using the new "in this guide" I suggested above, add an explicit "This is the real power of unbundled development, and the secret behind what makes Snowpack so fast."